tests/handlers/test_register.py

```diff
@@ -147,3 +144,30 @@ def test_register_saml2_mau_blocked(self):
         )
         with self.assertRaises(ResourceLimitError):
             yield self.handler.register_saml2(localpart="local_part")
+
+    @defer.inlineCallbacks
+    def test_auto_create_auto_join_rooms(self):
+        room_alias_str = "#room:test"
+        self.hs.config.autocreate_auto_join_rooms = True
```

Comment:
this seems to be the default; you could argue it should be set here for certainty but in that case let's have a comment. else just remove it